newIDE/app/src/UI/SoundPlayer/PlayButton.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+
+import * as React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import Play from '../CustomSvgIcons/Play';
+import Pause from '../CustomSvgIcons/Pause';
+import { makeStyles } from '@material-ui/styles';
+import GDevelopThemeContext from '../Theme/GDevelopThemeContext';
+
+type Props = {|
+  primary?: boolean,
+  isPlaying: boolean,
+  onClick: () => void,
+|};
+
+const PlayButton = (props: Props) => {
+  const gdevelopTheme = React.useContext(GDevelopThemeContext);
+
+  const classes = makeStyles({
+    root: {
+      color:
+        gdevelopTheme.soundPlayer.playButton[
```

Comment:
you can probably define `playButtonStyles = gdevelopTheme.soundPlayer.playButton[props.primary ? 'primary' : 'secondary']` to re-use everywhere

Reply:
Yes thanks!